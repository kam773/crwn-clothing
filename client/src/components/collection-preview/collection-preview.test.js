import React from 'react';
import { shallow, mount } from 'enzyme';
import { CollectionPreview } from './collection-preview.component';
import {
  PreviewContainer,
  TitleContainer,
} from './collection-preview.styles';

const mockProps = {
  title: 'title',
  items: [1, 2, 3, 4],
  routeName: 'routeName',
};

describe('create snapshot', () => {
  it('captures snapshot of CollectionPreview', () => {
    const wrapper = shallow(<CollectionPreview {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
  it('renders title', () => {
    const wrapper = shallow(<CollectionPreview {...mockProps} />);
    expect(wrapper.find(TitleContainer)).toBeDefined();
  });
  it('renders 4 collection items', () => {
    const wrapper = shallow(<CollectionPreview {...mockProps} />);
    expect(wrapper.find(PreviewContainer).children()).toHaveLength(4);
  });
});
