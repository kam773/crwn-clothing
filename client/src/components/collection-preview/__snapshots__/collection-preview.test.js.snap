// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collection-preview tests renders correctly collection-preview component 1`] = `ShallowWrapper {}`;

exports[`create snapshot captures snapshot of CollectionPreview 1`] = `ShallowWrapper {}`;
